frontend/src/lib/email/ForgotPassword.svelte

```diff
@@ -9,4 +9,5 @@
 <Email subject={$t('emails.forgot_password.subject')} {name}>
   <mj-text>{$t('emails.forgot_password.to_reset_click')}</mj-text>
   <mj-button href={resetUrl}>{$t('emails.forgot_password.reset_button')}</mj-button>
+  <mj-text>{$t('emails.link_expiration_warning')}</mj-text>
```

Comment:
It looks to me like the forgot-password jwt currently expires after only 1 hour.

Reply:
Maybe it's supposed to expire after only 1 hour, but the SendForgotPasswordEmail method currently sets the `useEmailLifetime` parameter to `true` so it uses the same 3-day expiration as everything else.

Update: I just checked with @hahn-kev and he says the forgot-password email token should expire in 3 days like other email tokens.

Reply:
Oh, you're right. Forgot-password is the only email that doesn't reference `useEmailLifetime` as a named parameter. So, I blindly thought it wasn't setting it to true.